scene/animation/tween_interpolaters.cpp

```diff
@@ -36,277 +36,277 @@ const real_t pi = 3.1415926535898;
+static real_t in_out(real_t time, real_t beginning, real_t change, real_t duration) {
+	if (time == 0) return beginning;
+	if (time == duration) return beginning + change;
+	time = time / duration * 2;
+	if (time < 1) return change / 2 * pow(2, 10 * (time - 1)) + beginning - change * 0.0005;
+	return change / 2 * 1.0005 * (-pow(2, -10 * (time - 1)) + 2) + beginning;
 }
 
-static real_t out_in(real_t t, real_t b, real_t c, real_t d) {
-	return (t < d / 2) ? out(t * 2, b, c / 2, d) : in((t * 2) - d, b + c / 2, c / 2, d);
+static real_t out_in(real_t time, real_t beginning, real_t change, real_t duration) {
+	return (time < duration / 2) ? out(time * 2, beginning, change / 2, duration) : in((time * 2) - duration, beginning + change / 2, change / 2, duration);
 }
 }; // namespace expo
 ///////////////////////////////////////////////////////////////////////////
 // elastic
 ///////////////////////////////////////////////////////////////////////////
 namespace elastic {
-static real_t in(real_t t, real_t b, real_t c, real_t d) {
-	if (t == 0) return b;
-	if ((t /= d) == 1) return b + c;
-	float p = d * 0.3f;
-	float a = c;
+static real_t in(real_t time, real_t beginning, real_t change, real_t duration) {
+	if (time == 0) return beginning;
+	if ((time /= duration) == 1) return beginning + change;
+	float p = duration * 0.3f;
+	float a = change;
 	float s = p / 4;
-	float postFix = a * pow(2, 10 * (t -= 1)); // this is a fix, again, with post-increment operators
-	return -(postFix * sin((t * d - s) * (2 * pi) / p)) + b;
+	float postFix = a * pow(2, 10 * (time -= 1)); // this is a fix, again, with post-increment operators
+	return -(postFix * sin((time * duration - s) * (2 * pi) / p)) + beginning;
 }
 
-static real_t out(real_t t, real_t b, real_t c, real_t d) {
-	if (t == 0) return b;
-	if ((t /= d) == 1) return b + c;
-	float p = d * 0.3f;
-	float a = c;
+static real_t out(real_t time, real_t beginning, real_t change, real_t duration) {
+	if (time == 0) return beginning;
+	if ((time /= duration) == 1) return beginning + change;
+	float p = duration * 0.3f;
+	float a = change;
 	float s = p / 4;
-	return (a * pow(2, -10 * t) * sin((t * d - s) * (2 * pi) / p) + c + b);
+	return (a * pow(2, -10 * time) * sin((time * duration - s) * (2 * pi) / p) + change + beginning);
 }
 
-static real_t in_out(real_t t, real_t b, real_t c, real_t d) {
-	if (t == 0) return b;
-	if ((t /= d / 2) == 2) return b + c;
-	float p = d * (0.3f * 1.5f);
-	float a = c;
+static real_t in_out(real_t time, real_t beginning, real_t change, real_t duration) {
+	if (time == 0) return beginning;
+	if ((time /= duration / 2) == 2) return beginning + change;
+	float p = duration * (0.3f * 1.5f);
+	float a = change;
 	float s = p / 4;
 
-	if (t < 1) {
-		float postFix = a * pow(2, 10 * (t -= 1)); // postIncrement is evil
-		return -0.5f * (postFix * sin((t * d - s) * (2 * pi) / p)) + b;
+	if (time < 1) {
+		float postFix = a * pow(2, 10 * (time -= 1)); // postIncrement is evil
```

Comment:
The whole point of the "improve readability" was to get rid of these inline operations like `time -= 1` or `time /= duration`.

Replacing all variable names without changing that is IMO pointless, it's not more readable with long variables names than with well documented short variable names (they have the same meaning in all methods, so t, b, c d is fine by me - what's not is these too clever one liners).

Reply:
The first and mandatory step was to change the names of the variables. 
Getting rid of these inline operations may be beneficial too.
But at least, this PR covers the first step.